packages/flex-plugins-utils-env/src/lib/__tests__/env.test.ts

```diff
@@ -42,18 +42,18 @@ describe('env', () => {
   });
 
   describe('isTerminalPersisted', () => {
-    it('should return true', () => {
+    it('isTerminalPersisted should return true', () => {
```

Comment:
how come you changed all of these? i believe the original code is the testing convention

Reply:
sonarlint comment about duplicate strings :p

Reply:
o lol 💩